alerting/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -5,4 +5,28 @@ grant {
 
     permission java.net.SocketPermission "*", "connect,resolve";
     permission java.net.NetPermission "getProxySelector";
+
+    // needed because of problems in ClientConfiguration
+    // TODO: get these fixed in aws sdk
+    permission java.lang.RuntimePermission "accessDeclaredMembers";
```

Comment:
Do we need `accessDeclaredMembers`? Could you please double check.

Reply:
It is required. The AWS SDK uses jackson databind, which needs `accessDeclaredMembers`.